solidity/contracts/bridge/Bridge.sol

```diff
@@ -209,6 +210,7 @@ contract Bridge is
         uint32 movingFundsTimeout,
         uint96 movingFundsTimeoutSlashingAmount,
         uint32 movingFundsTimeoutNotifierRewardMultiplier,
+        uint16 movingFundsCommitmentGasOffset,
```

Comment:
I think this should be enough, but just to point out that we limit ourselves to 65535 here.

Reply:
Yeah. I think this limit is enough. No offset is higher than 27k in `MaintainerProxy` now and we do not have a call to a separate contract in `Bridge` when committing to target wallets.